numpy/ma/core.py

```diff
@@ -4356,12 +4357,12 @@ def imag(self):
 
         See Also
         --------
-        get_real, real, imag
+        real, get_real, get_imag
```

Comment:
No point see-also-ing the other getters and setters - `real` alone should be fine here